index.html

```diff
@@ -109,6 +109,18 @@ <h2 class="f3">Courses</h2>
     </div>
   </div>
 
+  <div class="mv3 pa3 bg-light-gray">
```

Comment:
I think the element is a bit too prominent. Should we maybe instead highlight the news more to make it clear which posts are for studies (e.g. with some emoji)?

Reply:
I made it less prominent by removing the background and using the `cta` class instead of a button. I think we should keep the banner though.

Reply:
Okay. Let's keep it.